packages/components/src/menu-group/test/index.js

```diff
@@ -10,18 +10,29 @@ import { MenuGroup } from '../';
 
 describe( 'MenuGroup', () => {
 	test( 'should render null when no children provided', () => {
-		const wrapper = shallow( <MenuGroup /> );
+		render( <MenuGroup /> );
 
-		expect( wrapper.html() ).toBe( null );
+		expect( screen.queryByRole( 'group' ) ).not.toBeInTheDocument();
 	} );
 
-	test( 'should match snapshot', () => {
-		const wrapper = shallow(
-			<MenuGroup label="My group" instanceId="1">
+	test( 'should render children', () => {
+		render(
+			<MenuGroup>
 				<p>My item</p>
 			</MenuGroup>
 		);
 
-		expect( wrapper ).toMatchSnapshot();
+		expect( screen.queryByRole( 'group' ) ).toBeVisible();
+		expect( screen.queryByText( 'My item' ) ).toBeVisible();
+	} );
+
+	test( 'should render with an accessible label', () => {
+		render( <MenuGroup label="My group">Example</MenuGroup> );
+
+		expect(
+			screen.queryByRole( 'group', {
+				label: 'My group',
```

Comment:
I think we mean:

```suggestion
				name: 'My group',
```

(If you want, I think we can convert this test file to a `.tsx` and it will warn about these RTL mishaps.)